lib/ublksrv_aio.c

```diff
@@ -1,6 +1,9 @@
 // SPDX-License-Identifier: MIT or LGPL-2.1-only
 
+#ifndef _GNU_SOURCE
 #define _GNU_SOURCE
```

Comment:
Actually we take care of defining `_GNU_SOURCE` through configure workflow/config.h. So this redefinition looks unnecessary. I was contemplating if we really need explicit `#define _GNU_SOURCE` here. Wrapping it under `#ifndef _GNU_SOURCE`  felt safer, but would like your opinion in case I misunderstood the need.